<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsWindow</class>
 <widget class="QDialog" name="StatisticsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="mac_1">
    <attribute name="title">
     <string>MAC Endpoints</string>
    </attribute>
    <widget class="QTableView" name="tv_mac_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mac_2">
    <attribute name="title">
     <string>MAC Conversations</string>
    </attribute>
    <widget class="QTableView" name="tv_mac_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ip_1">
    <attribute name="title">
     <string>IP Endpoints</string>
    </attribute>
    <widget class="QTableView" name="tv_ip_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ip_2">
    <attribute name="title">
     <string>IP Conversations</string>
    </attribute>
    <widget class="QTableView" name="tv_ip_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>981</width>
       <height>551</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
